python/lsst/ts/scheduler/scheduler_csc.py

```diff
@@ -208,6 +216,12 @@ def __init__(
         # The maximum tolerable time without targets in seconds.
         self.max_time_no_target = 3600.0
 
+        # The maximum number of targets in the predicted schedule.
+        self.max_predicted_targets = 1000
+
+        # Default command response timeout
+        self.default_command_timeout = 60.0
+
         # How long to wait for target loop to stop before killing it
```

Comment:
Not part of the code changes but please end this sentence with `in seconds.` as well.